tests/xmlrpcapi/conftest.py

```diff
@@ -47,7 +56,11 @@ def cobbler_xmlrpc_base(cobbler_api: CobblerAPI) -> Tuple[CobblerXMLRPCInterface
     token = remote.login("", shared_secret)  # type: ignore
     if not token:
         sys.exit(1)
-    return remote, token
+    shared_secret2 = get_shared_secret()
+    token2 = remote.login("", shared_secret2)  # type: ignore
+    if not token2:
+        sys.exit(1)
+    return remote, token, token2
```

Comment:
There is only one shared secret. I wouldn't choose this approach. When an additional token is needed for a test let's write a fixture that uses the base fixtures, calls login and gives you a token aka a token generator. This way the fixture behavior stays stable. Also, the naming here is horrible IMHO.